tests/test_core.py

```diff
@@ -8754,6 +8791,8 @@ def setUp(self):
 wasm3 = make_run('wasm3', emcc_args=['-O3'])
 wasms = make_run('wasms', emcc_args=['-Os'])
 wasmz = make_run('wasmz', emcc_args=['-Oz'])
+# wasm64 = make_run('wasm64', emcc_args=['-O0', '-g3'],
+#                   settings={'MEMORY64':2}, env=None, node_args='--experimental-wasm-bigint')
```

Comment:
This doesn't need to be commented out like this.. it won't be run anywhere by default.